bootstrap.el

```diff
@@ -1,32 +0,0 @@
-;;; bootstrap.el --- Bootstrap quelpa.el
-
-;; Copyright 2014-2018, Steckerhalter
-
-;; Author: steckerhalter
-;; URL: https://framagit.org/steckerhalter/quelpa
-;; Version: 0.0.1
-;; License: https://framagit.org/steckerhalter/quelpa/LICENSE
-
-;; This file is not part of GNU Emacs.
-
-;;; Code:
-
-(require 'package)
```

Comment:
Deleting this file is breaking change. Lot of user who doesn't use Melpa will have to update their script to not use `bootstrap.el`. Perhaps you may want to update README or not delete this file

Reply:
Thanks, I misunderstood and thought it was only used for testing.

I would prefer to document that users should "bootstrap" manually by copying `quelpa.el` into their Emacs config rather than running `bootstrap.el`.  Is that okay with you, or do you prefer to keep it?

Reply:
Copying `quelpa.el` works for me too. In that case, please also document it as it's breaking change. I would imagine that a lot of people that's using `bootstrap.el` suddenly got it stop working.
Nit: here is how I configure my quelpa from custom source. Perhaps we can adapt it and replace the part of using `bootstrap.el` in README file with that?
``` elisp
(unless (package-installed-p 'quelpa)
    (with-temp-buffer
      (url-insert-file-contents "https://raw.githubusercontent.com/quelpa/quelpa/master/quelpa.el")
      (eval-buffer)
      (quelpa-self-upgrade)))
```